README.md

```diff
@@ -16,8 +16,8 @@ This content helps you bring your solution to the Microsoft marketplace faster a
 1. [The SaaS Accelerator](https://microsoft.github.io/Mastering-the-Marketplace/saas-accelerator)
 1. [Azure Managed Application Offers](https://microsoft.github.io/Mastering-the-Marketplace/ama)
 1. [Using Partner Center](https://microsoft.github.io/Mastering-the-Marketplace/partner-center)
+1. [Metered Billing Accelerator](https://microsoft.github.io/Mastering-the-Marketplace/metered-billing-accelerator)
 
-You will need to clone this repository to do the hands-on labs including in the courses. 
 
```

Comment:
Good idea to remove this from the main page, but has it been added back within the SaaS Accelerator? Just so users know that they should clone those sections to do the respective labs to use scripts etc